toolkit/docs/nvidia/mariner-nvidia.repo

```diff
@@ -0,0 +1,9 @@
+[mariner-official-nvidia]
```

Comment:
Please also update the documentation about available .repo files under `toolkit/docs/building/building.md`. Search for "REPO_LIST=...".

Reply:
This is not required in this scenario.